### PR TITLE
Feature #162

### PR DESCRIPTION
**Pull request by bmoscon**
See #162 - Adds a rename method to ChunkStore to allow symbol renaming.

### CODE REVIEW

**Comment by bmoscon** [comment]
@jamesblackburn

**Comment by bmoscon** [comment]
@jamesblackburn I had to change the travis and circle scripts to update Mongo. The default version is some ancient 2.4 version. I'll squash the merge :)

**Comment by bmoscon** [comment]
@jamesblackburn note that the rename method only works on newer versions of Mongo. Not sure if thats important or not. Based on some things  I read online it looks like there was some bug in older versions that was fixed in 2.6. I could change the code to still support older versions of Mongo, or we could say we only support newer versions than 2.4 (or we could say everything except rename works in 2.4). Or something else?

File: arctic/chunkstore/chunkstore.py [hunk 1]
```diff
@@ -126,6 +126,44 @@ def list_symbols(self):
...
+
+        chunks = []
+        for x in self._collection.find({SYMBOL: from_symbol}, sort=[(START, pymongo.ASCENDING)],):
+            chunks.append(x)
+
+        bulk = self._collection.initialize_unordered_bulk_op()
+        for chunk in chunks:
+            chunk.pop('_id', None)
+            chunk[SYMBOL] = to_symbol
+            chunk[SHA] = self._checksum(to_symbol, chunk)
```
Comment (jamesblackburn) [inline arctic/chunkstore/chunkstore.py]: Could be optimised to remove the symbol from the SHA, then this turns into a simple `update` operation.

**Comment by jamesblackburn** [comment]
LGTM!

**Pull request merged**